fixtures/sub-process-no-killable

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
```

Comment:
You should run `xo` on this file. I am not sure why `npm test` does not do it automatically. But `xo` seems to complain about the whitespace before `console.log()`, and the lack of an uppercase letter at the start of the comment `run forever`.

Reply:
Mh... running xo directly on this odesn't seem to help (no output).
I tried to fix it though: c0c3527d06777617a30b6e4391442f5a5eef3244

Perhaps xo excludes fixtures by default? (Dropping the fixtures  patch in the package.json didnt help).